tests/test_apps/run_dir_test/my_app.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import time
+from pathlib import Path
+from typing import Tuple
+
+from omegaconf import DictConfig
+
+import hydra
+from hydra.core.hydra_config import HydraConfig
+from hydra.utils import get_original_cwd
+
+
+@hydra.main(config_name="config")
+def my_app(_: DictConfig) -> None:
+    run_dir = Path.cwd().relative_to(get_original_cwd())
+    time.sleep(5)
```

Comment:
Why do we need a sleep here?

Reply:
Context: https://github.com/facebookresearch/hydra/issues/840